readlif/utilities.py

```diff
@@ -1,6 +1,9 @@
-from readlif.reader import _check_magic, _check_mem, _read_int
 import xml.etree.ElementTree as ET
 
+from bs4 import BeautifulSoup
+
+from readlif.reader import _check_magic, _check_mem, _read_int
```

Comment:
Not for this PR, but I find it confusing that a utils module imports from the main module, and that private functions are being used outside their module scope.

Reply:
Feel free to open a PR 😉